core-bundle/src/Resources/contao/modules/ModuleChangePassword.php

```diff
@@ -66,7 +67,7 @@ protected function compile()
 
 		$this->import(FrontendUser::class, 'User');
 
-		$GLOBALS['TL_LANGUAGE'] = $objPage->language;
+		$GLOBALS['TL_LANGUAGE'] = LocaleUtil::formatAsLanguageTag($objPage->language);
```

Comment:
Why are we overriding `$GLOBALS['TL_LANGUAGE']` here at all? It is set in `PageRegular::prepare()` and randomly overridden in `ModuleChangePassword`, `ModulePassword`, `ModulePersonalData` and `ModuleRegistration` but not in any other module. 🤔

Reply:
I would guess because in oder versions we did set `$GLOBALS['TL_LANGUAGE']` when loading the user, see https://github.com/contao/contao/commit/4aeef258e62498a4fc2a06d0f7cc9a9da785d20e#diff-5f45d2efbaa881457d499dd868b6666e6fed8cf8cd8c5471f4a3735dd05f7666L381-L399

So this is problably a left over, and can be removed. Because now it seems that we set the language on login and not when the user is loaded.

Reply:
@aschempp Do you agree as well?

Reply:
I don't consider this part of this PR. I think we agreed a while ago to keep this for BC and because it didn't harm ever.

Reply:
Agreed. See #3154